src/coreclr/jit/fginline.cpp

```diff
@@ -609,8 +609,11 @@ class SubstitutePlaceholdersAndDevirtualizeWalker : public GenTreeVisitor<Substi
                                                 isLateDevirtualization, explicitTailCall);
                 if (context != nullptr)
                 {
-                    IL_OFFSET ilOffset = BAD_IL_OFFSET;
-                    INDEBUG(ilOffset = call->gtRawILOffset);
+#if defined(DEBUG)
+                    IL_OFFSET ilOffset = call->gtRawILOffset;
+#else
+                    IL_OFFSET ilOffset = m_curStmt->GetDebugInfo().GetLocation().GetOffset();
```

Comment:
Debug information cannot be used as a substitute for exact IL offsets. However, it appears `InlineCandidateInfo::ilOffset` is unused, so we can just remove the field and argument to `impMarkInlineCandidate`.